Sources/PayPal/Models/PayPalRequest.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+/// Used to configure options for approving a PayPal order
+public struct PayPalRequest {
```

Comment:
Hey there!
Nice Idea to follow the CardRequest pattern 👌

I got a question, why using a struct and not a protocol to define what should be a PayPalRequest ?

Thanks 🙇‍♂️

Reply:
While we could certainly use a protocol for the request and have the struct conform to it, the lack of reuse (at least currently) for the protocol as well as the current simplicity of the objects have lead us to use structs for the request objects. This also is a similar pattern to what Apple uses when constructing a `PKPaymentRequest` for Apple Pay. That said, if we see a strong need to move in a protocol direction in the future we would certainly explore this option.

Reply:
Thanks a lot @jaxdesmarais for your answer 🙏